scripts/db/src/get-user.ts

```diff
@@ -50,48 +64,100 @@ async function getByEmail(email: string, callback: DbScriptCallback) {
       POSTGRES_HOST,
       POSTGRES_DATABASE,
       POSTGRES_PORT,
+      MONGO_URI,
+      MONGO_DB_NAME,
     } = (configuration as unknown) as DbConfiguration
 
-    /** Declare connection info */
-    const connectionInfo: ConnectionConfig = {
-      user: POSTGRES_USERNAME,
-      password: POSTGRES_PASSWORD,
-      host: POSTGRES_HOST,
-      database: POSTGRES_DATABASE,
-      port: POSTGRES_PORT ? parseInt(POSTGRES_PORT) : 5432,
-      ssl: TEMPLATE_DATA.pgShouldSsl,
-    }
+    /**
+     * Logic in this function tries to match:
+     * EAForum/packages/lesswrong/server/vulcan-lib/apollo-server/authentication.tsx
+     */
+    async function getForumUser(): Promise<CallbackUser | null> {
+      // Connect to the Forum DB
+      const mongoClient: MongoClient = new MongoClient(MONGO_URI)
+      await mongoClient.connect()
+
+      // Query the users collection for someone with our email
+      const forumUser = await mongoClient
+        .db(MONGO_DB_NAME)
+        .collection<ForumUser>('users')
+        .findOne({ 'emails.address': email })
```

Comment:
Are emails stored case-sensitively? If so, maybe `email.toLowerCase()`, or use some Mongo query magic that searches case insensitively?

Reply:
I'm taking this directly from LW, which uses case-sensitive emails. I think this is probably bad, but don't think we should fix it here.